webapp/lib/view.dart

```diff
@@ -36,7 +37,13 @@ void init() {
   replyPanelView = new ReplyPanelView();
   tagPanelView = new TagPanelView();
   authHeaderView = new AuthHeaderView();
-  authMainView = new AuthMainView();
+  authMainView = new AuthMainView(
+    "assets/africas-voices-logo.svg",
+    "Welcome to Nook.", 
+    "Sign in to Nook where you can manage SMS conversations.",
+    [SignInDomain.avf, SignInDomain.lark, SignInDomain.ucam, SignInDomain.gmail],
```

Comment:
I think the others snuck in from one of the Cambridge collaboration projects - as the default we should only have AVF and Lark to match the firestore rules.
```suggestion
    [SignInDomain.avf, SignInDomain.lark],
```